app/src/main/java/at/bitfire/icsdroid/ui/EditCalendarActivity.kt

```diff
@@ -153,7 +153,7 @@ class EditCalendarActivity: AppCompatActivity() {
         model.active.value = calendar.isSynced
 
         val (username, password) = CalendarCredentials(this).get(calendar)
-        val requiresAuth = username != null && password != null
+        val requiresAuth = !username.isNullOrEmpty() && !password.isNullOrEmpty()
```

Comment:
Hm does that mean that username/password are empty instead of null when there is no authentication?